app/models/label/TagTable.scala

```diff
@@ -8,15 +8,16 @@ import play.api.cache.Cache
 import scala.concurrent.duration.DurationInt
 import scala.slick.lifted.{ForeignKeyQuery, Index}
 
-case class Tag(tagId: Int, labelTypeId: Int, tag: String, mutuallyExclusiveWith: Option[String])
+case class Tag(tagId: Int, labelTypeId: Int, tag: String, mutuallyExclusiveWith: Option[String], count: Int = 0)
```

Comment:
It's a bit annoying, but let's keep the `Tag` class just containing the data in the table. We can just create a new case class if need be. For every table we have a case class with exactly that data, so I don't think it's worth it to break that convention.